src/website/components/Header.tsx

```diff
@@ -0,0 +1,115 @@
+import React, { PropsWithChildren, ReactElement } from "react";
+import { Link } from "react-router-dom";
+import styled from "styled-components";
+
+import { AicsLogoAndNameSVG, AicsLogoSVG } from "../assets/icons";
+import { FlexRowAlignCenter } from "./LandingPage/utils";
+
+const AICS_LOGO_RESIZE_THRESHOLD_PX = 540;
+/**
+ * Used to determine header height on viewer page.
+ * TODO: Determine this dynamically?
+ */
+export const HEADER_HEIGHT_PX = 61;
+
+const AicsLogoLink = styled.a`
+  position: relative;
+  width: 140px;
+  height: 36px;
+
+  div > svg:last-child {
+    display: none;
+  }
+
+  // Toggle between the two logos based on the currently available screen real estate
+  // Width is determined here experimentally to prevent popping as the other buttons in the header wrap.
+  @media only screen and (max-width: ${AICS_LOGO_RESIZE_THRESHOLD_PX}px) {
+    & {
+      max-width: 36px;
+      max-height: 36px;
+    }
+
+    & > div > svg:first-child {
+      display: none;
+    }
+
+    & > div > svg:last-child {
+      display: block;
+      visibility: visible;
+    }
+  }
+`;
+
```

Comment:
There is a mid-width where the logos switch over and the load and share buttons seem to get mis-positioned, as the screen size drops lower they go back to where they are supposed to be.
<img width="463" alt="Screenshot 2024-04-16 at 11 58 42 AM" src="https://github.com/allen-cell-animated/website-3d-cell-viewer/assets/24981838/ae3cb5ad-f426-4b4e-be36-c41a3de431da">

Reply:
Yup, I need to tweak the magic numbers a bit for this viewer :X 
I haven't added the help dropdown yet either so I'll fix it for now but add a TODO.

Reply:
Fixed!